source/guides/maintenance/custom-app-container.rst

```diff
@@ -0,0 +1,402 @@
+
+	 # Grab os-release info from the minimal base image so
+	 # that the new content matches the exact OS version
+	-COPY --from=clearlinux/os-core:latest /usr/lib/os-release /
+	+COPY --from=clearlinux/os-core:30550 /usr/lib/os-release /
+
+	 # Install additional content in a target directory
+	 # using the os version from the minimal base
+	@@ -26,7 +26,7 @@ COPY --from=clearlinux/os-core:latest / /os_core_install/
+	 RUN cd / && \
+	     find os_core_install | sed -e 's/os_core_install/install_root/' | xargs rm -d &> /dev/null || true
+
+	-FROM clearlinux/os-core:latest
+	+FROM clearlinux/os-core:30550
+	 MAINTAINER qi.zheng@intel.com
+
+	 COPY --from=builder /install_root /
+
+
+	$ docker build --no-cache=true --build-arg http_proxy=$http_proxy --build-arg https_proxy=$https_proxy -t clearlinux/httpd:30550 httpd/
+
+
+	diff --git a/cgit/Dockerfile b/cgit/Dockerfile
+	index 9a3796d..59260fe 100644
+	--- a/cgit/Dockerfile
+	+++ b/cgit/Dockerfile
+	@@ -7,7 +7,7 @@ RUN swupd update --no-boot-update $swupd_args
+
+	 # Grab os-release info from the minimal base image so
+	 # that the new content matches the exact OS version
+	-COPY --from=clearlinux/httpd:latest /usr/lib/os-release /
+	+COPY --from=clearlinux/httpd:30550 /usr/lib/os-release /
+
+	 # Install additional content in a target directory
+	 # using the os version from the minimal base
+	@@ -22,11 +22,11 @@ RUN source /os-release && \
+	 # file exists on different layers. To minimize docker
+	 # image size, remove the overlapped files before copy.
+	 RUN mkdir /os_core_install
+	-COPY --from=clearlinux/httpd:latest / /os_core_install/
+	+COPY --from=clearlinux/httpd:30550 / /os_core_install/
+	 RUN cd / && \
+	     find os_core_install | sed -e 's/os_core_install/install_root/' | xargs rm -d &> /dev/null || true
+
+	-FROM clearlinux/httpd:latest
+	+FROM clearlinux/httpd:30550
+
+	 COPY --from=builder /install_root /
+	 COPY cgitrc /etc/cgitrc
+
+
+	$ docker build --no-cache=true --build-arg http_proxy=$http_proxy --build-arg https_proxy=$https_proxy -t clearlinux/cgit:30550 cgit/
+
+
+Once the target container is built, the installed OS version can be verified
+by noting the :command:`VERSION_ID` value in the :file:`/usr/lib/os-release`
+file in the container filesystem.
+
+.. code-block:: bash
+
+	$ docker run clearlinux/cgit:30550 cat /usr/lib/os-release
+	NAME="Clear Linux OS"
+	VERSION=1
+	ID=clear-linux-os
+	ID_LIKE=clear-linux-os
+	VERSION_ID=30550
+	PRETTY_NAME="Clear Linux OS"
+	ANSI_COLOR="1;35"
+	HOME_URL="https://clearlinux.org"
+	SUPPORT_URL="https://clearlinux.org"
+	BUG_REPORT_URL="mailto:dev@lists.clearlinux.org"
+	PRIVACY_POLICY_URL=http://www.intel.com/privacy
+
+
+Customize an application image at runtime
+=========================================
+
+Besides generating a new Docker image by editing the Dockerfile, you can
+modify a published |CL| container at runtime. This approach can
+help to accelerate the feature development process. This section describes

```

Comment:
```suggestion
be faster to get started. This section describes
```

Reply:
Revise para. 
This section describes how to modify a published |CL| container at runtime.  In this example, we add Tensorflow\* to the `:command:clearlinux/python` container.  This approach can help accelerate the feature development process.